### DISCUSSION TITLE
Incorrect Hashrate and Suspicious Account Notifications

### DISCUSSION THREAD

**Discussion by Cravati** [revoxhere/duino-coin#1603] [Q&A]
I keep getting the incorrect hashrate and suspicious account notifications. Running a Raspberry Pi 3b hosting 9x Arduino Nano's. Raspbian is updated. Miners are running versions 3.33. The only issue I notice is that the website only shows 8 of the 9 miners. 
![Screenshot_20230121-214539](https://user-images.githubusercontent.com/90291521/213901199-01ea847f-f683-44ea-a8ad-ef0b461271f9.png)
![PXL_20230122_044130416](https://user-images.githubusercontent.com/90291521/213901206-3eb66358-8247-4ef5-9abd-ddfd8f5e0041.jpg)
![PXL_20230122_044136684](https://user-images.githubusercontent.com/90291521/213901210-fc672609-1d46-44a2-8e7b-b7c362d5e96a.jpg)

**Comment by 3Domse3** [comment]
Got the same issue with my Teensy 4.1 :(
